tutor/src/screens/question-library/exercises-display.jsx

```diff
@@ -106,6 +110,28 @@ class ExercisesDisplay extends React.Component {
 
   scroller = new Scroller({ windowImpl: this.windowImpl });
 
+  componentWillUnmount() {
+    this.props.exercises.clear();
+    document.removeEventListener('scroll', this.checkIsControlsSticky);
+  }
+
+  componentDidMount() {
+    document.addEventListener('scroll', this.checkIsControlsSticky);
+  }
+
```

Comment:
if we add event listeners to `scroll`, its important to debounce the callback so it doesn't block scrolling and lead to jittery behavior. In fact the latest chrome will yell at you for it.  

We have a observable model for this, https://github.com/openstax/tutor-js/blob/main/tutor/src/models/window-scroll.js   Use it instead.